checker/src/org/checkerframework/checker/units/Math.astub

```diff
@@ -0,0 +1,146 @@
+import org.checkerframework.checker.units.qual.*;
+
+package java.lang;
+
+// https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html
+class Math {
+    // Constants
+    static final @Scalar double E;
+    static final @Scalar double PI;
+
+    // Methods ========
+
+    // Absolute Val
+    static @PolyUnit double abs(@PolyUnit double a);    // Same ret
+    static @PolyUnit float abs(@PolyUnit float a);    // Same ret
+    static @PolyUnit int abs(@PolyUnit int a);    // Same ret
+    static @PolyUnit long abs(@PolyUnit long a);    // Same ret
+
+    // Trigonometry
+    static @Scalar double sin(@radians double a);
+    static @Scalar double cos(@radians double a);   // cos
+    static @Scalar double tan(@radians double a);
+    // both x & y must be same unit and if so returns that unit, else returns LUB of the units
+    static @PolyUnit double hypot(@PolyUnit double x, @PolyUnit double y);
+
+    // Inverse Trigonometry
+    static @radians double asin(@Scalar double a);   // arc-sin
+    static @radians double acos(@Scalar double a);   // arc-cos
+    static @radians double atan(@Scalar double a);   // arc-tangent
+    // x and y should be the same unit, returns the angle theta from the conversion of rectangular coordinates [x, y] to polar coordinates [r, theta]. returns radians
```

Comment:
Some lines in here are overly long, break them properly.
Instead of "after-the-signature" comments I prefer "line-before-signature" comments.